### PR TITLE
Forms 828 : The exports doesn't give good render for iterations

### PR DESCRIPTION
**Pull request by Louis-Edmond**

### CODE REVIEW

**Comment by Louis-Edmond** [comment]
I had to make changes on IEntryDataService. So in order to not break compatibility i added i new method instead of modifying the one used before the changes.

**Comment by Louis-Edmond** [comment]
I don't understand why tis pull request included some old commits from other tickets though. I don't know how to fix this

**Comment by rzara** [comment]
Did you make your change on top of the latest develop ?
When creating your pull request, did you choose develop as base ?

File: src/java/fr/paris/lutece/plugins/forms/web/entrytype/IEntryDataService.java [hunk 1]
```diff
@@ -90,7 +90,16 @@
...
+    List<String> responseToStrings( FormQuestionResponse formQuestionResponse );
+    
+    /**
+     * Extracts the responses from the specified form question response as a map of {@code String} objects
+     * 
+     * @param formQuestionResponse
+     *            the response to convert
+     * @return the Map of {@code String} objects
+     */
+    Map<Integer, List<String>> responseToIterationsStrings(FormQuestionResponse formQuestionResponse);
```
Comment (rzara) [inline src/java/fr/paris/lutece/plugins/forms/web/entrytype/IEntryDataService.java:102]: you are still breaking backward compatibility by adding a new method without a default implementation. Maybe you should bring the implementation up in this interface as default ? see https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html

Comment (Louis-Edmond) [reply src/java/fr/paris/lutece/plugins/forms/web/entrytype/IEntryDataService.java:102]: Before working on my ticket, i create a new local branch (named like the related jira ticket) based on my develop branch ( i regularely check if my develop branch need to be updated by pulling).
When my work is done and i want to push my changes i check if there is any new changes from remote branch develop. If it's the case i switch to my develop branch, pull, then switch back to my ticket branch (in this case FORMS-828) and merge develop to it. Then i push my local branch FORMS-828 on my remote FORMS-828 branch (on my personal plugin-forms repo).
Then i create my new pull request by comparing my FORMS-828 branch with the develop remote branch.

Comment (rzara) [reply src/java/fr/paris/lutece/plugins/forms/web/entrytype/IEntryDataService.java:102]: you should probably rebase instead of merging

**Comment by Louis-Edmond** [comment]
I realized i could just write my method on a another class so i don't have to use IEntryDataService. So i changed back so this class remain unchanged.

**Comment by Louis-Edmond** [comment]
recreating a cleaner pull request without unrealated commits

**Pull request closed without merging**